tests/cephfs/snapshot_clone/clone_attributes.py

```diff
@@ -150,10 +246,57 @@ def run(ceph_cluster, **kw):
         log.info("Clean Up in progess")
         rmclone_list = [
             {"vol_name": default_fs, "subvol_name": "clone_attr_vol_1"},
+            {"vol_name": default_fs, "subvol_name": "clone_attr_vol_2"},
         ]
         for clone_vol in rmclone_list:
             fs_util.remove_subvolume(client1, **clone_vol)
-        fs_util.remove_snapshot(client1, **snapshot, validate=False, check_ec=False)
+        fs_util.remove_snapshot(client1, **snapshot_1, validate=False, check_ec=False)
+        fs_util.remove_snapshot(client1, **snapshot_2, validate=False, check_ec=False)
         fs_util.remove_subvolume(client1, **subvolume, validate=False, check_ec=False)
         for subvolumegroup in subvolumegroup_list:
             fs_util.remove_subvolumegroup(client1, **subvolumegroup, force=True)
+
+
+def compare_json(subvol_info_1, subvol_info_2, keys_to_check=None):
+    """
+    Compares specified keys in two JSON strings for equality.
+
+    Args:
+        subvol_info_1 (str): First JSON string.
+        subvol_info_2 (str): Second JSON string.
+        keys_to_check (list): List of keys to compare. Defaults to a predefined set of keys.
+
+    Returns:
+        tuple: (bool, dict) where the bool indicates if all keys match,
+               and dict contains mismatched keys with their respective values.
+    """
+    # Default keys to check
+    default_keys = [
+        "atime",
+        "bytes_quota",
+        "bytes_used",
+        "data_pool",
+        "features",
+        "flavor",
+        "gid",
+        "mode",
+        "mon_addrs",
+        "uid",
+    ]
+    keys_to_check = keys_to_check or default_keys
+
+    # Load JSON data
+
+    mismatches = {}
+
+    for key in keys_to_check:
+        if subvol_info_1.get(key) != subvol_info_2.get(key):
+            mismatches[key] = {
+                "value1": subvol_info_1.get(key),
+                "value2": subvol_info_2.get(key),
+            }
+
+    # Return True if no mismatches, otherwise return mismatches
+    if not len(mismatches) == 0:
+        raise CommandFailed(f"clone attributes are mismatching {mismatches}")
+    # return (len(mismatches) == 0), mismatches
```

Comment:
This function does not have return statement. I think missed to uncomment

Reply:
We are raising a CommandFailed exception to intentionally fail the test case. Since this function is exclusively used for this specific test, I have commented out the return statement and raised the exception instead.

Reply:
Merging this for now as the doc strings update is the only pending one. Please make a note and fix it in the next PR @AmarnatReddy